src/Raven.Server/RavenServer.cs

```diff
@@ -276,24 +276,30 @@ public void RefreshClusterCertificate(object state)
         {
             try
             {
-                CertificateHolder newCertificate = null;
+                CertificateHolder newCertificate;
                 try
                 {
                     newCertificate = LoadCertificate();
+                    if (newCertificate == null)
+                    {
+                        if (Logger.IsOperationsEnabled)
+                            Logger.Operations("Tried to load certificate as part of refresh check, and got a null back, but got a valid certificate on startup!");
+
+                        ServerStore.NotificationCenter.Add(AlertRaised.Create(
+                            null,
+                            "Server certificate",
+                            "Tried to load certificate as part of refresh check, and got a null back, but got a valid certificate on startup!",
```

Comment:
might be good idea to use a single local variable for duplicated "Tried to load certificate as part of refresh ...." message so we won't need to keep them in sync if we change the message text